src/Centreon/Domain/Filter/FilterService.php

```diff
@@ -111,6 +111,15 @@ public function addFilter(Filter $filter): int
      */
     public function updateFilter(Filter $filter): void
     {
+        $foundFilter = $this->filterRepository->findFilterByUserIdAndName(
```

Comment:
you could add a unit test to check this case :)

Reply:
Done 👍